### PR TITLE
BLO-27 Added unit tests.

### PR DESCRIPTION
**Pull request by RikvanSpreuwel**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// module.exports = {
```
Comment (hugovandevliert) [inline jest.config.js]: Remove this commented code

File: junit.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hugovandevliert) [inline junit.xml]: This file should not be added to git

File: src/app.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+export class App {
+    private readonly port: number;
+    private readonly routers: AbstractRouter[];
+
+    constructor(routers: AbstractRouter[], port: number) {
+        this.routers = routers;
+        this.port = port;
+
+        // this.initSentry();
```
Comment (hugovandevliert) [inline src/app.ts]: Why is sentry not enabled?

**Review by hugovandevliert — APPROVED** [review]

**Review by hugovandevliert — DISMISSED** [review]
Please disregard my previous approved review, that happened by accident.

File: src/app.ts (hunk 3 above)
Comment (Svenmarim) [reply src/app.ts]: Already fixed in newly commits :)

**Review by roelvdboom — APPROVED** [review]

**Pull request merged**